components/NavigationDrawer/src/MDCBottomDrawerPresentationController.h

```diff
@@ -31,6 +31,21 @@
 - (void)bottomDrawerWillChangeState:
             (nonnull MDCBottomDrawerPresentationController *)presentationController
                         drawerState:(MDCBottomDrawerState)drawerState;
+
+/**
+ This method is called when the drawer is scrolled/dragged and provides a transition ratio value
```

Comment:
_Tip:_ What does 100% mean and what does 0% mean, I think adding those would be easier to understand for future users. `100% is content is full screen and 0% is content is in it's default state`